elements/pfe-codeblock/README.md

```diff
@@ -0,0 +1,72 @@
+# PatternFly Element | Codeblock element
+Render code in a styled and fromatted way
+
+## Usage
+Describe how best to use this web component along with best practices.
+
+```html
+<pfe-codeblock>
+    <!-- Default slot -->
+    <h2>This is pfe-codeblock</h2>
+    
+</pfe-codeblock>
+```
+
+### Accessibility
+Explain how this component meets accessibility standards.
+
+## Slots
+
+- `namedSlot`: Describe each available slot and best practices around what markup it can contain.
+
+## Attributes
```

Comment:
The attributes below need to be updated.

- code-language: 
- code-line-numbers:
- code-line-number-start:
- code-theme: